packages/@aws-cdk/aws-ecs/lib/fargate/fargate-service.ts

```diff
@@ -73,15 +80,31 @@ export interface IFargateService extends IService {
 export class FargateService extends BaseService implements IFargateService {
 
   /**
-   * Import a task definition from the specified task definition ARN.
+   * Import a service definition from the specified task definition ARN.
    */
   public static fromFargateServiceArn(scope: cdk.Construct, id: string, fargateServiceArn: string): IFargateService {
+    const matcher = new RegExp(/arn:[^:]+:ecs:[^:]+[^:]+:[^:]+:service\/(.*)/);
     class Import extends cdk.Resource implements IFargateService {
       public readonly serviceArn = fargateServiceArn;
+      public readonly serviceName = matcher.exec(fargateServiceArn)[0];
     }
     return new Import(scope, id);
   }
 
+  /**
+   * Import a service definition from the specified service attributes.
+   */
+  public static fromFargateServiceAttributes(scope: cdk.Construct, id: string, attributes: string): IBaseService {
+    const matcher = new RegExp(/arn:[^:]+:ecs:[^:]+[^:]+:[^:]+:service\/(.*)/);
+    class Import extends cdk.Resource implements IBaseService {
+      public readonly cluster = ;
+      public readonly serviceArn = fargateServiceArn;
+      public readonly serviceName = matcher.exec(fargateServiceArn)[0];
+    }
+    return new Import(scope, id);
+  }
+
+
   /**
    * Constructs a new instance of the FargateService class.
    */
```

Comment:
Same exact comments in this file as in `ec2-service.ts` (use `Stack.parseArn`, new interface `FargateServiceAttributes`).